dogu.json

```diff
@@ -90,6 +90,24 @@
       "Validation": {
         "Type": "BINARY_MEASUREMENT"
       }
```

Comment:
The keys for memory and swap limit are missing.

Reply:
The values were already inside the dogu.json for a preceding release